File: J2465/JS/Aug/21-08/Task FA3.40/scripts/product.js
const _name = Symbol("name");
const _price = Symbol("price");
const _category = Symbol("category");

class Product {
  constructor(name, price, category) {
    this[_name] = name;
    this[_price] = price;
    this[_category] = category;
  }

  getInfo() {
    return `Name: ${this[_name]}, Price: ${this[_price]}, Category: ${this[_category]}`;
  }
}

export { Product };

File: J2465/JAVA/Nov/21-11/Task STR8.10/demo/src/main/java/com/example/demo/model/Product.java
package com.example.demo.model;

public class Product {
    public String name;
    public String category;
    public Double price;

    public Product(String name, String category, Double price) {
        this.name = name;
        this.category = category;
        this.price = price;
    }

    @Override
    public String toString() {
        return "Product [name=" + name + ", category=" + category + ", price=" + price + "]";
    }

}
